src/redux/reducers/parentReducer.js

```diff
@@ -187,6 +190,84 @@ export const initialState = {
       sender_id: 1,
     },
   ],
+  bookings: {
+    isFetching: false,
+    bookings: [
```

Comment:
Not totally necessary but if you want to reduce the size of lines inside this reducer you could define this dummy data elsewhere and import in.